Gemfile

```diff
@@ -108,4 +105,6 @@ group :test do
   # for test coverage report
   gem 'simplecov', require: false
 
+  # for controller testing
+  gem 'rails-controller-testing'
```

Comment:
@neerajdotname Just checked [this comment by DHH](https://github.com/rails/rails/issues/18950#issue-57746192). It makes sense to remove `assigns` and `assert_template`.